CIP-1694/README.md

```diff
@@ -535,7 +553,7 @@ The following table details the ratification requirements for each governance ac
 | Governance action type                                            | CC   | DReps    | SPOs     |
 | :---                                                              | :--- | :---     | :---     |
 | 1. Motion of no-confidence                                        | \-   | $P_1$    | $Q_1$    |
-| 2<sub>a</sub>. New committee/threshold (_normal state_)           | ✓    | $P_{2a}$ | \-       |
+| 2<sub>a</sub>. New committee/threshold (_normal state_)           | \-   | $P_{2a}$ | $Q_{2a}$ |
```

Comment:
Now that we've shifted the Governance Action's "New Committee/Threshold," where both the normal state and the state of no-confidence require the same parties (DReps and SPOs) to vote, I'm wondering if we need to differentiate between them and list both in the table separately. I think we might not need separate thresholds for both states of the GA, as voting for a new committee or threshold should always be the same.

What do you think? Am I missing something?

Reply:
> I think we might not need separate thresholds for both states of the GA

I was not sure about this myself, that's exactly why I left two rows.